drivers/sub8_videoray_m5_thruster/nodes/thruster_driver.py

```diff
@@ -183,6 +183,10 @@ def publish_bus_voltage(self, *args):
 
     def check_bus_voltage(self, voltage):
         ''' Raises bus_voltage alarm with a corresponding severity given a bus voltage '''
+        # Timing bug occurs here occasionally so I (David) wil disable enforcing bus_voltage kills
```

Comment:
No need for attribution here. I'm sure we will all be running `git blame` when problems arise, don't you worry... Also, there's a spelling error on this line.